CRVResponse/src/CrvWaveformsGenerator_module.cc

```diff
@@ -70,6 +78,10 @@ namespace mu2e
     EDProducer{pset},
     _crvSiPMChargesModuleLabel(pset.get<std::string>("crvSiPMChargesModuleLabel")),
     _singlePEWaveformFileName(pset.get<std::string>("singlePEWaveformFileName")),
+    _eventWindowMarkerLabel(pset.get<std::string>("eventWindowMarker","EWMProducer")),
```

Comment:
Do these window times need to be synchronized with the parameters with the same name in CrvResponse?  If so, can you think of a way to coordinate these?  Maybe through DaqConditions?

Reply:
Yes, they need to be the same everywhere. I will change it to get the information from the conditions.